tools/code_format/check_format_test_helper.py

```diff
@@ -7,17 +7,21 @@
 
 from __future__ import print_function
 
-from run_command import runCommand
 import argparse
 import logging
 import os
 import shutil
 import sys
 import tempfile
 
-tools = os.path.dirname(os.path.realpath(__file__))
+curr_dir = os.path.dirname(os.path.realpath(__file__))
+tools = os.path.dirname(curr_dir)
+# due to dependency on run_command
+sys.path.append(tools)
+from run_command import runCommand
```

Comment:
This feels very hacky and brittle.. is there a better way to address the fact that there is a dependency on `run_command.py`, which lives in `tools`?

Reply:
Yeah agreed this seems not great, but my Python foo is weak and @htuch is out right now. @lizan any idea on best practice here? One idea that comes to mind would be to somehow use a venv to put together all of the deps that are needed? I'm not sure what is best.

Reply:
@mattklein123 WDYT about having a `check_format_test_helper.sh` wrapper around `check_format_test_helper.py` that sets the `PYTHONPATH` env variable so that it lists the `tools` dir? IMHO a virtualenv might be a bit much in this case, since the only dep we have is the `run_command.py` in `tools`.

Reply:
Sure sounds great.

Reply:
Addresses this in https://github.com/envoyproxy/envoy/pull/9978/commits/252898201e5d1212d6a7173eddeace6597dbf87b